alt-tab-macos/api-wrappers/PrivateApis.swift

```diff
@@ -0,0 +1,11 @@
+import Cocoa
+import Foundation
+
+class PrivateApis {
+    // CGS can capture screenshots of minimized windows, which CGWindowListCreateImage can't
+    static func windowScreenshot(_ windowId: inout UInt32) -> CGImage {
```

Comment:
maybe rename to `minimizedWindowScreenshot` to make the purpose clear without the need for the comment?

Reply:
This method can capture screenshots of any window, not just minimized. In this PR i only use this method for all windows as a matter of fact